std_msgs/Header header
bool safe_to_move
string reason
builtin_interfaces/Time last_event_time
uint32 robot_mode
uint32 robot_state
bool emergency_stop
bool protective_stop
bool[] joint_limit_violations